src/code/utils/get-query-param.coffee

```diff
@@ -0,0 +1,9 @@
+module.exports = (param) ->
+  param = param.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]")
+  regexS = "[\\?&]" + param + "=([^&#]*)"
+  regex = new RegExp regexS
+  results = regex.exec window.location.href
+  if results?.length > 1
+    return decodeURIComponent results[1]
+  else
+    return null
```

Comment:
The day that coffeescript supports ternary operators will be a day that I rejoice! Until then...

(In any case, even if it supported it, it's just a style question, not a logic simplification. And I actually like having `return` statements be fairly explicit, particularly in if-blocks, even when CS returns things implicitly.)